### PR TITLE
Update monero-core_nl.ts

### PR DESCRIPTION
**Pull request by ElectricSlacks**
Gone from 83 type="unfinished"s to 37.

### CODE REVIEW

**Comment by ElectricSlacks** [comment]
Ref #1582

File: translations/monero-core_nl.ts [hunk 1]
```diff
@@ -982,7 +982,7 @@
...
-        <translation type="unfinished"></translation>
+        a<translation type="unfinished">&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Dit is een simpele verkoop traceerder:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Laat je klant de QR code scannen om een betaling te make (Als de klant software heeft die QR code scannen ondersteund).&lt;/p&gt;&lt;p&gt;Deze pagina scant de blockchain en de transactiepool voor inkomende transacties die gebruik maakten van deze QR code. Als je een aantal invult dan zal het ook de transacties controleren alsof ze totalen naar dat aantal.&lt;/p&gt;&lt;p&gt;It&apos;s Alsof je onbevestigde transactie wil accepteren of niet. It is likely they&apos;ll be confirmed in short order, but there is still a possibility they might not, so for larger values you may want to wait for one or more confirmation(s).&lt;/p&gt;</translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Last sentence not translated.

File: translations/monero-core_nl.ts [hunk 2]
```diff
@@ -1047,37 +1047,37 @@
...
         <location filename="../pages/settings/SettingsInfo.qml" line="118"/>
         <source>Wallet path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Portemonnee pad:</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="155"/>
         <source>Wallet creation height: </source>
-        <translation type="unfinished"></translation>
+        <translation>Portemonnee creëer hoogte:</translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: If you translate this literally, it should be one word without spaces. 
Has previously been translated as `Hoogte waarop portemonnee is gemaakt:`

File: translations/monero-core_nl.ts [hunk 3]
```diff
@@ -1204,7 +1209,7 @@ The old wallet cache file will be renamed and can be restored later.
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="399"/>
         <source>Stop local node</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Stop lokale </translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Missing word: `Stop lokale node`.

File: translations/monero-core_nl.ts [hunk 4]
```diff
@@ -1214,110 +1219,110 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/settings/SettingsNode.qml" line="421"/>
         <source>(default)</source>
-        <translation type="unfinished"></translation>
+        <translation>(standaard)</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="449"/>
         <source>Daemon startup flags</source>
-        <translation type="unfinished"></translation>
+        <translation>Daemon opstart commando's</translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Previously translated as `Startparameters voor lokale node`.
Flags are not commands.

File: translations/monero-core_nl.ts [hunk 5]
```diff
@@ -1214,110 +1219,110 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/settings/SettingsWallet.qml" line="276"/>
         <source>Store this information safely to recover your wallet in the future.</source>
-        <translation type="unfinished"></translation>
+        <translation>Bewaar deze informatie veilig om je portemonnee te herstellen in de toekomst.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsWallet.qml" line="311"/>
         <source>Show seed</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Laat </translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Missing words.

File: translations/monero-core_nl.ts [hunk 6]
```diff
@@ -1214,110 +1219,110 @@ The old wallet cache file will be renamed and can be restored later.
...
         <location filename="../pages/settings/SettingsWallet.qml" line="416"/>
         <source>Information</source>
-        <translation type="unfinished">Informatie</translation>
+        <translation>Informatie</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsWallet.qml" line="417"/>
         <source>Successfully rescanned spent outputs.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Succesvol uitgaven</translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Missing words. Previously translated as `Uitgegeven outputs zijn opnieuw opgezocht.`

File: translations/monero-core_nl.ts [hunk 7]
```diff
@@ -2376,17 +2381,17 @@ Voor een betalingsbewijs hoeft u het adres van de ontvanger niet op te geven.</t
...
         <location filename="../wizard/WizardManageWalletUI.qml" line="340"/>
         <source>Device name</source>
-        <translation type="unfinished"></translation>
+        <translation>Apparaatnaam</translation>
     </message>
     <message>
         <location filename="../wizard/WizardManageWalletUI.qml" line="344"/>
         <source>Ledger</source>
-        <translation type="unfinished"></translation>
+        <translation>Kasboek</translation>
```
Comment (ProkhorZ) [inline translations/monero-core_nl.ts]: Context is device name.

**Comment by erciccione** [comment]
@ProkhorZ What do you feel about working together with @ElectricSlacks on this? maybe you could be added as collaborator on his branch and commit your changes, so that @ElectricSlacks's work won't be discharged. The commit could be co-authored, so to don't lose anybody's attribution.

Would be also good to have some more pair of eyes on this work. @fridzema @potatored Could you guys give a look to this PR?

**Comment by ElectricSlacks** [comment]
Hi all.

First of all i want to apologize for my sub par work so far. I would like
to help very much. So if we could divide work between the 2/ (how many
people would work on this) and co author this.
That would be very nice!

Greetings from ElectricSlacks

Op do 4 okt. 2018 14:48 schreef erciccione <notifications@github.com>:

**Comment by ProkhorZ** [comment]
> What do you feel about working together with @ElectricSlacks on this?

No, I expect some mistakes in amateur translations, and I've learned from other translators in the past, but this was worse than useless. Worse than machine translation. I only pointed out some of the most conspicuous issues, because there were errors in literally more than half of the strings. And the translation wasn't even complete. 

I'm not going to cooperate with someone who is sloppy to the extent that words are missing in multiple strings. I don't care about the attribution. 

@ElectricSlacks: Apology accepted. The only way I can help you is to make this a learning experience.

**Comment by ElectricSlacks** [comment]
Hey i get it. it is the worst so for using this on Monero would be very stupid. For learning purposes i would like to work further on this. And learn how to do this.
Best would be to close this PR and open a new one.

**Comment by erciccione** [comment]
> Best would be to close this PR and open a new one.

@ElectricSlacks then please close this PR, so that sombody else (@ProkhorZ ?) can open a new one :)

In the meantime, i suggest you to read the guide the @ProkhorZ himself did to help monero translators. You can find it here: https://github.com/monero-ecosystem/monero-translations/blob/master/translation-tips.md

**Pull request closed without merging**